src/pages/ReimbursementAccount/CompanyStep.js

```diff
@@ -189,106 +153,93 @@ class CompanyStep extends React.Component {
                     />
                     <View style={styles.mt4}>
                         <Picker
+                            inputID="incorporationType"
                             label={this.props.translate('companyStep.companyType')}
                             items={_.map(this.props.translate('companyStep.incorporationTypes'), (label, value) => ({value, label}))}
-                            onInputChange={value => this.clearErrorAndSetValue('incorporationType', value)}
-                            value={this.state.incorporationType}
                             placeholder={{value: '', label: '-'}}
-                            errorText={this.getErrorText('incorporationType')}
+                            defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'incorporationType')}
+                            shouldSaveDraft
                         />
                     </View>
                     <View style={styles.mt4}>
                         <DatePicker
+                            inputID="incorporationDate"
                             label={this.props.translate('companyStep.incorporationDate')}
-                            onInputChange={value => this.clearErrorAndSetValue('incorporationDate', value)}
-                            defaultValue={this.state.incorporationDate}
                             placeholder={this.props.translate('companyStep.incorporationDatePlaceholder')}
-                            errorText={this.getErrorText('incorporationDate')}
                             maximumDate={new Date()}
+                            defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'incorporationDate')}
+                            shouldSaveDraft
                         />
                     </View>
                     <View style={styles.mt4}>
                         <StatePicker
+                            inputID="incorporationState"
                             label={this.props.translate('companyStep.incorporationState')}
-                            onInputChange={value => this.clearErrorAndSetValue('incorporationState', value)}
-                            value={this.state.incorporationState}
-                            errorText={this.getErrorText('incorporationState')}
+                            defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'incorporationState')}
+                            shouldSaveDraft
                         />
                     </View>
                     <CheckboxWithLabel
-                        isChecked={this.state.hasNoConnectionToCannabis}
-                        onInputChange={value => this.clearErrorAndSetValue('hasNoConnectionToCannabis', value)}
+                        inputID="hasNoConnectionToCannabis"
+                        defaultValue={!!ReimbursementAccountUtils.getDefaultStateForField(this.props, 'hasNoConnectionToCannabis')}
```

Comment:
I think that we can remove the double ! by passing `false` as default value to `getDefaultStateForField`

```suggestion
                        defaultValue={ReimbursementAccountUtils.getDefaultStateForField(this.props, 'hasNoConnectionToCannabis', false)}
```

Reply:
done